### PR TITLE
Add option to ignore color palette in TIFF images

### PR DESCRIPTION
**Pull request by gnodar01**
`getImageFromIFD` will automatically create a 16 bit RGB if there if a tiff is of type 3. This makes sense most times, but occasionally, we need the raw single channel greyscale image.

### CODE REVIEW

File: index.d.ts [hunk 1]
```diff
@@ -32,7 +32,7 @@ export declare class Image {
   static createFrom(other: Image, options: ImageConstructorOptions): Image;
   static load(
     image: string | ArrayBuffer | Uint8Array,
-    options?: RequestInit,
+    options?: RequestInit | { ignorePalette: boolean },
```
Comment (targos) [inline index.d.ts:35]:
```suggestion
    options?: RequestInit & { ignorePalette: boolean },
```

**Review by targos — COMMENTED** [review]
Can you please add a test to https://github.com/image-js/image-js/blob/legacy/src/image/core/__tests__/loadTIFF.js ?

**Comment by gnodar01** [comment]
> Can you please add a test to https://github.com/image-js/image-js/blob/legacy/src/image/core/__tests__/loadTIFF.js ?

Done. Not sure I understand what's going on with the code coverage, but if there's something I can do just let me know. Thanks!

**Comment by targos** [comment]
Thanks!

**Pull request merged**